support/Python/Schedule.py

```diff
@@ -110,9 +111,18 @@ def _copy_submit_script_template(
     return dest
 
 
-# Write `pathlib.Path` objects to YAML as plain strings
-def _path_representer(dumper: yaml.Dumper, path: Path) -> yaml.nodes.ScalarNode:
-    return dumper.represent_scalar("tag:yaml.org,2002:str", str(path))
+# Write following objects to YAML
+#  - `pathlib.Path`
+#  - np.float64
+def _path_representer(dumper: yaml.Dumper, to_dump) -> yaml.nodes.ScalarNode:
+    if isinstance(to_dump, Path):
```

Comment:
add a separate function for `np.float64` instead of these conditionals. The function is then added with `add_multi_representer` below.